sample_annotator/geolocation/geotools.py

```diff
@@ -36,10 +36,26 @@ def get_client(self):
             self.client = googlemaps.Client(key=self.googlemaps_api_key)
         return self.client
 
-    def get_elevation(self, latlon: LATLON = (40.714224, -73.961452)):
-        results = self.get_client().elevation(latlon)
-
-        return results
+    def get_elevation(self, latlon: LATLON) -> str:
+        elevprefixurl = 'https://webmap.ornl.gov/ogcbroker/wms?originator=QAQCIdentify&SERVICE=WMS&VERSION=1.1.1&REQUEST=GetFeatureInfo&SRS=EPSG:4326&WIDTH=5&HEIGHT=5&LAYERS=10003_1&QUERY_LAYERS=10003_1&X=2&Y=2&INFO_FORMAT=text/xml&BBOX='
```

Comment:
I think it would be better if the query parameters could be encoded as key/value pairs.  
See #42

Reply:
Updated for query params as requested